### PR TITLE
[REVIEW][Doc] Fixes a list of minor errors in the example codes #457

### PR DESCRIPTION
**Pull request by sosae0**
fixes https://github.com/rapidsai/cusignal/issues/457

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Comment by awthomp** [comment]
ok to test

File: python/cusignal/convolution/convolve.py [hunk 1]
```diff
@@ -219,7 +219,7 @@ def fftconvolve(in1, in2, mode="full", axes=None):
     >>> fig, (ax_orig, ax_mag) = plt.subplots(2, 1)
     >>> ax_orig.plot(cp.asnumpy(sig))
     >>> ax_orig.set_title('White noise')
-    >>> ax_mag.plot(np.arange(-len(sig)+1,len(sig)), autocorr)
+    >>> ax_mag.plot(cp.asnumpy(cp.arange(-len(sig)+1,len(sig))), cp.asnumpy(autocorr))
```
Comment (awthomp) [inline python/cusignal/convolution/convolve.py:222]: We're getting a Python Style warning here. Do you mind changing this line from:

```python
ax_mag.plot(cp.asnumpy(cp.arange(-len(sig)+1,len(sig))), cp.asnumpy(autocorr))
```

to

```python
ax_mag.plot(cp.asnumpy(cp.arange(-len(sig)+1,len(sig))),
    cp.asnumpy(autocorr))
```

You can install flake8 into your conda environment and run:

`flake8 convolve.py` as well to ensure our coding style checks pass.

**Review by awthomp — CHANGES_REQUESTED** [review]
Minor nit to fix flake8 CI warning. Otherwise, looks good!

To update this PR, simply make the fix *in your branch* and then commit and push to your local, forked, feature branch. This will automatically update the PR here.

**Review by awthomp — APPROVED** [review]

**Comment by awthomp** [comment]
@gpucibot merge

**Comment by sosae0** [comment]
My pleasure @awthomp. Thanks for the support.

**Pull request merged**